fmslack/cli.py

```diff
@@ -114,9 +115,11 @@ def slack_post(slack_webhook_url, name, artist, album, image):
 
     logger.debug('Posting to Slack webhook {0}'.format(slack_webhook_url))
 
+    artists = ' & '.join([artist['name'] for artist in artists])
```

Comment:
List comprehensions (along with dict comprehensions) are one of the awesome features of python and very powerful when combined with other batteries included features, like string joining as we are doing here.